charts/k8tz/templates/certificate.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.webhook.certManager.enabled -}}
+{{- $fqdn := printf "%s.%s.svc" (include "k8tz.serviceName" .) .Values.namespace }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
```

Comment:
once the certificated will be rotated (90 days), cert-manager will know to restart k8tz controller pod as well?

remember that the controller [mounts](https://github.com/k8tz/k8tz/blob/master/charts/k8tz/templates/deployment.yaml#L26) the certificate from the secret. if certificate is expired and pod not restarted - it will cause tls validation errors.

Reply:
I'll do some test with short certificate retention time, and observe how cert-manager handle this

Reply:
just created `Certificate` with
```
spec:
  duration: 1h
  renewBefore: 59m
```
after few minutes, webhook starts failing the requests

Reply:
guess this is a common problem when using cert-manager to handle certs.
see https://github.com/cert-manager/cert-manager/issues/1440 https://github.com/Kong/kubernetes-ingress-controller/issues/986

Reply:
yes i see, cert-manager consider this problem as out of their scope.
if the cluster have `wave` installed, it will be handled by wave controller. but we can't assume that, at least not by default.
maybe the best we can do is to set a long expiration as we do in a regular installation ([5114 days](https://github.com/k8tz/k8tz/blob/139d9af48153a590ef667ef1fed658a6e34e1ba1/charts/k8tz/templates/admission-webhook.yaml#L2)) and let the user configure that to a shorter time if he have wave (or similar solution) installed.
currently, keeping the default of 90 days expiration is a time bomb.

Reply:
oh okay then, I'll add the `duration` and `renewBefore` in the `Certificate` template

Reply:
do you have any suggestion for `renewBefore` time field? can we set it like 1d?

Reply:
we are talking about ~14 years certificate, so let's be honest it doesn't really matter :smile:, but maybe better that `renewBefore` will be more than 1 day. let's say 30 days...

Reply:
sure, will update it